internal/pkg/admission/policy-server-deployment.go

```diff
@@ -205,19 +205,19 @@ func (r *Reconciler) deployment(configMapVersion string, policyServer *policiesv
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      policyServer.NameWithPrefix(),
 			Namespace: r.DeploymentsNamespace,
-			Labels:    constants.PolicyServerLabels,
+			Labels:    policyServer.AppLabel(),
```

Comment:
This is probably a personal preference, I'd do something like:

```suggestion
			Labels:    map[string]string{
			  constants.AppLabel: policyServer.AppLabel(),
			}
```

The `AppLabel` function would return the `string` only. I think is easier to add more labels if needed in the future and to keep them separate, but this is just bike shedding :)

Reply:
make sense, that way it would be easier to add labels in the future. I'll change it in the next commit